lib/screens/user_registration/widgets/password_feild.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+
+class PasswordField extends StatefulWidget {
```

Comment:
you can reuse the PasswordField widget in login/widgets/ to avoid duplication of code

Reply:
Should I do it for Confirm Password as well?